docs/js-core-3/week-1/mentors.md

```diff
@@ -70,100 +78,106 @@ You should reserve 10-15 minutes at the end of the lesson to run through the fol
 
 #### Solutions
 
-1. Possible answers:
+##### Question 1
```

Comment:
Auto-formatter re-arranged this file so I've reformatted it using headings to fix it